src/libs/EmojiUtils.js

```diff
@@ -0,0 +1,76 @@
+/* eslint radix: ["error", "as-needed"] */
```

Comment:
I think we can just add the radix here.. is it something other than `10`? That's what we do everywhere else.

Reply:
Well, with Emojis you can’t be sure that the you’ll get an integer with base 10. This could be because copy pasting emojis from other apps. 

If you see the code, we are splitting a string and it could start with 0x too.